packages/back-end/src/services/auth/index.ts

```diff
@@ -125,7 +125,7 @@ export async function processJWT(
     } else {
       checkProjects = [project];
     }
-    for (const p in checkProjects) {
+    for (const p of checkProjects) {
```

Comment:
`checkProjects` is an array such as [ 'projectId1', 'projectId2', ...]. `for...in` will actually iterate with values of index like 0, 1, 2, ... , `for...of` should be used instead.